tests/keras/layers/convolutional_test.py

```diff
@@ -487,27 +487,61 @@ def test_zero_padding_3d():
     stack_size = 2
     input_len_dim1 = 4
     input_len_dim2 = 5
-    input_len_dim3 = 3
+    input_len_dim3 = 13
```

Comment:
Use a smaller dim to reduce the time taken by the test (it was already an expensive test, now with 4x bigger inputs and more test cases it is even more expensive).